src/rgw/rgw_lua_utils.h

```diff
@@ -201,7 +204,11 @@ int StringMapWriteableNewIndex(lua_State* L) {
 
   const char* index = luaL_checkstring(L, 2);
   const char* value = luaL_checkstring(L, 3);
-  map->insert_or_assign(index, value);
+  if (strlen(value) < MAX_LUA_VALUE_SIZE && map->size() < MAX_LUA_KEY_ENTRIES) {
+    map->insert_or_assign(index, value);
+  } else {
+    return luaL_error(L, "Lua table size limit exceeded");
```

Comment:
there are 2 different errors:
"max number of entries exceeded"
"maximum size of entry exceeded"